### DISCUSSION TITLE
[Feature] Add to album and archive [or] auto-archive by album

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#3096] [Feature Request]
### The feature

I find myself adding large number of certain types of photos to albums that I don't also want showing in my main photos display.  I keep thinking it would be nice if I didn't have to

1) select all the photos
2) add to album
3) select all the photos AGAIN
4) archive.

Ideally, to me, would be a per-album option where anything that's added to that album is also archived.

Alternatively would be an "Add to Album and Archive" menu item.

Thanks.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by colin715** [comment]
I would really appreciate an option to archive assets from the album view.  Many times, I am adding photos from various points in the timeline to an album.  In the current workflow, I would then have to find them all again in the main timeline, hoping I don't miss any, then archive.

**Reply by jrasm91** [reply]
In the meantime, I'll add the action to the overflow menu (#3773).

**Comment by bo0tzz** [comment]
#1673
